python/xorbits/sklearn/datasets/tests/test_core.py

```diff
@@ -0,0 +1,45 @@
+# Copyright 2022-2023 XProbe Inc.
```

Comment:
Add a simple UT to test `sklearn.datasets`, not just testing doc.
Like the `test_kmeans_cluster` you added above. 
Same as other modules.

Reply:
tests are added except `neighbors`, related issue https://github.com/xorbitsai/xorbits/issues/739